cmd/command_utils.go

```diff
@@ -135,3 +143,37 @@ func maybeUpload(cloudCredentials, cloudStorageBucket, cloudProvider, path strin
 		cmdLogger.Error("Unknown cloud provider")
 	}
 }
+
+// writeParquet creates the parquet file and writes the exported data into it.
+//
+// Parameters:
+//
+//	data []transform.SchemaParquet  - The slice of data to be written to the Parquet file.
+//										SchemaParquet is an interface used to call ToParquet()
+//										which is defined for each schema/export.
+//	path string                     - The file path where the Parquet file will be created and written.
+//										For example, "some/file/path/export_output.parquet"
+//	schema interface{}              - The schema that defines the structure of the Parquet file.
+//
+//	Errors:
+//
+//	stellar-etl will log a Fatal error and stop in the case it cannot create or write to the parquet file
+func writeParquet(data []transform.SchemaParquet, path string, schema interface{}) {
+	parquetFile, err := local.NewLocalFileWriter(path)
+	if err != nil {
+		cmdLogger.Fatal("could not create parquet file: ", err)
+	}
+	defer parquetFile.Close()
+
+	writer, err := writer.NewParquetWriter(parquetFile, schema, 1)
+	if err != nil {
+		cmdLogger.Fatal("could not create parquet file writer: ", err)
+	}
+	defer writer.WriteStop()
+
+	for _, record := range data {
+		if err := writer.Write(record.ToParquet()); err != nil {
+			cmdLogger.Fatal("could not write record to parquet file: ", err)
+		}
+	}
+}
```

Comment:
A batch conversion would  have been so noice 😿

Reply:
Yeah it would have been but I didn't see an example of it. Only writing sequentially from this example
https://github.com/xitongsys/parquet-go/blob/master/example/writer.go#L42-L53

Although I didn't spend any time digging through the code to see if it was easy to implement

Reply:
Agree. I took a look as well. Could not find anything.